common/src/main/java/com/wynntils/core/config/upfixers/ConfigUpfixerManager.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.config.upfixers;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.config.upfixers.impl.CustomPoiVisbilityUpfixer;
+import com.wynntils.core.managers.Manager;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConfigUpfixerManager extends Manager {
+    public final String UPFIXER_JSON_MEMBER_NAME = "upfixers";
```

Comment:
Feels like a bit short as a json name. All our other json config keys are like `feature>.<something>.<some more>` or suchlike. Maybe `config.upfixers`, or `wynntils.upfixers` to sort of "claim" a namespace for more general stuff related to the mod/config file as a whole.